client/src/features/searchV2/components/SearchV2Results.tsx

```diff
@@ -116,60 +116,60 @@ function SearchV2ResultsContent() {
     return <SearchV2ResultsUnknown key={`unknown-result-${index}`} />;
   });
 
-  return <Row className="gy-4">{resultsOutput}</Row>;
+  return <Row className="g-3">{resultsOutput}</Row>;
 }
 
 interface SearchV2ResultsCardProps {
   children?: ReactNode;
 }
-function SearchV2ResultsCard({ children }: SearchV2ResultsCardProps) {
+function SearchV2ResultsContainer({ children }: SearchV2ResultsCardProps) {
   return (
-    <Col xs={12} lg={6}>
-      <Card className={cx("border", "rounded", "h-100")} data-cy="search-card">
-        <CardBody className={cx("d-flex", "flex-column")}>{children}</CardBody>
-      </Card>
+    <Col xs={12} lg={6} xxl={4} data-cy="search-card">
+      <Card className="h-100">{children}</Card>
     </Col>
   );
 }
 
 interface SearchV2CardTitleProps {
-  children?: ReactNode;
-  entityType: ReactNode;
-  url: string;
+  name: string;
+  nameUrl: string;
```

Comment:
```suggestion
  entityUrl: string;
```

as it is more informative.